code/mathematical-algorithms/Check_is_square/check_sqaure.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+#include <math.h>
+#include <stdbool.h>
+
+bool issqaure(int n)
+{
+	int sroot=sqrt(n);
+	if(sroot*sroot == n)
+		return true;
+
+	return false;
+}
+
+int main()
+{
+	printf("Enter the no. to be examined.\n");
+	double n;
```

Comment:
Since the `issquare` take `int` as its input, why should we use a `double` to get out input ?

Reply:
made the relevant changes.